packages/table/src/table.tsx

```diff
@@ -441,7 +443,9 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     // when true, we'll need to imperatively synchronize quadrant views after
     // the update. this variable lets us avoid expensively diff'ing columnWidths
     // and rowHeights in <TableQuadrantStack> on each update.
-    private didUpdateColumnOrRowSizes: boolean = false;
+    private didUpdateColumnOrRowSizes = false;
+
+    private hasCompletelyRendered = false;
```

Comment:
nit: above we use `did*` but this one uses `has*`. it's the same tense but we should probably use the same word for consistency

Reply:
Changed to `didCompletelyMount` (emphasis on _"mount"_) because its value toggles only when all cells finish mounting, not for each "_render_" cycle thereafter.